helm/values-dev.yaml

```diff
@@ -0,0 +1,28 @@
+namespace: dev-ep-bitswap-peer
+image:
+  version: "20220513.1754"
+service:
+  metrics:
+    port: 3001
+    publicEndpoint: "false"
+    enableMonitor: "false"
+resources:
+  limits:
+    cpu: 2
+    memory: 4Gi
+  requests:
+    cpu: 1
+    memory: 1Gi
+autoscaling:
+  enabled: true
+  minReplicas: 1
+  maxReplicas: 5
+  targetCPUUtilizationPercentage: 70
+  targetMemoryUtilizationPercentage: 70
+env:
+  nodeEnv: dev
+  peerIdS3Bucket: us-west-2-dev-ep-bitswap-config
+  concurrency: 128
+  pipelining: 16
+  cacheBlocksInfo: false
```

Comment:
Can't these duplicates be set in the base?

Reply:
Done

Reply:
Done!